README.md

```diff
@@ -190,10 +190,10 @@ Wikipedia users share the sum of free knowledge. They create the content collabo
 
 Even though the author is an Engineer at Wikimedia foundation, this is not an official Wikimedia foundation project.
 
-[1]: ipfs.io/
-[2]: datproject.org
+[1]: https://ipfs.io/
+[2]: https://datproject.org
```

Comment:
this is redirecting to https://dat.foundation/ . But I have not made the change.